site/class-1-play/maaikevc.svg

```diff
@@ -0,0 +1,35 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 700 700" width="500" height="500">
+ 
+  <title>maaikevc</title>
```

Comment:
Almost! Now just add an `@` before your username here (you can do that here on GitHub, by clicking on the pencil Icon to the right!)

Reply:
Yes, I added a @ before my name, is this ok or do I have to make another PR?